src/style.css

```
@@ -1 +1,108 @@
 /* add styles here */
+:root {
+--shadow: rgba(61, 78, 97, 0.10);
+--border-color: #E1E7ED;
+}
+
+body {
+  margin: 8px;
+  display: block;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+.search-form__input::placeholder {
+  color: #3D4E61;
+  font-family: Avenir , sans-serif;
```

apply font-family for `html` tag selector and let inputs inherit it 
```suggestion
  font-family: inherit;
```